SPECS/xenserver-release.spec

```diff
@@ -33,10 +33,12 @@ Obsoletes:      control-xscontainer = 1.0-1
 Obsoletes:      control-xenserver-measured-boot = 1.0-1
 Obsoletes:      control-pvsaccelerator = 1.0-1
 
-#Obsolete XS73 hotfixes
+#Obsolete XS73+XS74 hotfixes
 Obsoletes:      update-XS73 control-XS73
 Obsoletes:      update-XS73E001 control-XS73E001
 Obsoletes:      update-XS73E002 control-XS73E002
+Obsoletes:      update-XS73E003 control-XS73E003
+Obsoletes:      update-XS74 control-XS74
```

Comment:
FYI, this is related to XS patches using their "pack" system. Which is replaced by just a yum repo on our side.

Reply:
But it's still useful to ease upgrade from XenServer 7.4,with the patches applied, I suppose. That will make sure the hotfix packages are removed upon upgrade.